net.peercoin.peercoin-qt.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "net.peercoin.peercoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
```

Comment:
```suggestion
  "runtime-version": "20.08",
```

Reply:
For some reason I could not pull 20.08 locally so I've went with older version.

Reply:
19.08 will be EOL in a few weeks.

Reply:
Alright will bump it up.